build_test.go

```diff
@@ -1136,7 +1139,7 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 					ClearCache: true,
 					Buildpacks: []string{"missing.bp@version"},
 				}),
-					"invalid buildpack string 'missing.bp@version'",
+					"downloading buildpack: error reading missing.bp@version: invalid locator: InvalidLocator",
```

Comment:
I guess there is a slight change in that users would see a different error message. Not sure how important it is that our messaging stays strictly the same.